c-dependencies/js-compute-runtime/xqd.wit

```diff
@@ -193,7 +193,7 @@ default world xqd-world {
 
     http-req-close: func(h: request-handle) -> result<_, error>
 
-    enum content-encodings {
+    flags content-encodings {
```

Comment:
If I remember correctly from the `cache_override_tag` case, the first flag in WIT is a 1, while the first flag in WITX was a 0?

Either way though, it would be great if we could ensure we don't assume binary value alignment between the interfaces and actually ensure we're passing the value through a conversion function where it is used.

Reply:
The first flag will be the 0 bit, which is consistent with witx. Here we're always passing `0` for a content-encodings value, as that's what the first enum value translates to, which corresponds to a value with no flags set in viceroy.

Reply:
If you are sure. I just recall that when we first implemented the cache_tag flags @JakeChampion had to remove the `none` value to get these to align with WITX, which made me think that encoding might have been starting from 0 - https://github.com/fastly/Viceroy/blob/main/lib/compute-at-edge-abi/typenames.witx#L123.